src/ui_fsmenu/launch_game.cc

```diff
@@ -78,13 +78,13 @@ LaunchGame::LaunchGame(MenuCapsule& fsmm,
      win_condition_duration_(&right_column_content_box_,
                              0,
                              0,
-                             200,
+                             300,
                              200,
```

Comment:
Thank you.

I think that `unit_w` should be increased though:

800x600, w = 300, unit_w = 200 (current):

![duration_w200](https://user-images.githubusercontent.com/80712255/198874649-1c22e6b0-982b-404e-aa96-a8c6bbda1cfd.png)

800x600, w = 360, unit_w = 240:

![duration_w240](https://user-images.githubusercontent.com/80712255/198874651-df8d0b1c-6909-40bc-90ad-0136adcde48f.png)